modules/ci_environment/manifests/jenkins_job_support/postgresql.pp

```diff
@@ -10,4 +10,16 @@
       password_hash => postgresql_password('jenkins', 'jenkins'),
       createdb      => true;
   }
+
+  # For mapit
+  class { 'postgresql::server::postgis': }
+
+  # The mapit role needs to be a superuser in order to load the PostGIS
+  # extension for the test database.
+  postgresql::server::role {
+    'mapit':
+      superuser     => true,
+      password_hash => postgresql_password('mapit', 'mapit');
```

Comment:
We should put a password for this role into hieradata instead of having it here, since it's a superuser. We'll then need to make it accessible to the test run, so we could create an envvar for it and use that in CI-specific config in the app.